app/Resources/views/base.html.twig

```diff
@@ -88,6 +88,9 @@
 
         {% block footer %}
             <hr>
+            <ul id="footerLinks">
```

Comment:
I think that is the result of a lapse of PR judgement on my part. Let's do kebab case for both.